src/Module.zig

```diff
@@ -4103,6 +4103,21 @@ pub fn scanNamespace(
     for (decls) |decl_inst| {
         try scanDecl(&scan_decl_iter, decl_inst);
     }
+
+    if (seen_decls.count() != namespace.decls.count()) {
+        // Do a pass over the namespace contents and remove any decls from the last update
+        // which were removed in this one.
+        var i: usize = 0;
+        while (i < namespace.decls.count()) {
```

Comment:
missing `i += 1` ?

Reply:
4294 should make it unnecessary but 4295 should immediately crash

Reply:
Oops, you're totally right, good spot! Luckily this doesn't affect code built without incremental, and this is going to be changed a lot again soon anyway, but thank you for pointing out the issue.